src/API/TimesheetController.php

```diff
@@ -122,6 +128,13 @@ public function cgetAction(ParamFetcherInterface $paramFetcher)
             $query->setPageSize($size);
         }
 
+        if (null !== ($tags = $paramFetcher->get('tags'))) {
+            $dbResult = $this->tagRepository->findIdsByTagNameList($tags);
```

Comment:
Do you have any idea where I can implement this query? Before executing the timesheet-query it is important to load the tags which fit into the pattern (requested tags). Scrutinizer tells that the implementation could be a fault. So I could extend the TimesheetRepository. BUT it is not clean code if I use one RepositoryClass in another. Or do you have another look on this topic?

Reply:
Scrutinizer complains about a lot of places which are totally fine, I guess you can ignore that. 
I think for the first version its okay like it is. Even though, I would rename `$dbResult` to `$ids`.